classes/submission/reviewAssignment/maps/Schema.php

```diff
@@ -24,24 +25,26 @@ class Schema extends \PKP\core\maps\Schema
 
     public string $schema = PKPSchemaService::SCHEMA_REVIEW_ASSIGNMENT;
 
+    public Enumerable $submissions;
```

Comment:
AFAICS this property isn't shared nor reused in later calls, so I think it's better to demote it to a local variable.